tycho-core/pom.xml

```diff
@@ -258,13 +258,16 @@
 			<groupId>commons-io</groupId>
 			<artifactId>commons-io</artifactId>
 		</dependency>
-
 		<dependency>
 			<groupId>commons-codec</groupId>
 			<artifactId>commons-codec</artifactId>
 			<version>1.16.0</version>
 		</dependency>
-
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.12.0</version>
```

Comment:
For some reason this was nessary to avoid classloading Problems at runtime. But actually this is provided by Maven itself (and marked as provided Dependency of maven-core).

Reply:
Replaced the usage of StringUtils with standard java.